packages/core/src/Source.ts

```diff
@@ -130,6 +134,9 @@ export default class Source {
     }
     if (shouldInvokeAfterUpdate === true) {
       this.filesystem.clearCache();
+      this.filesystem.unfreeze();
+      await this.invokeAfterUpdate(sharedFilesystem, globalConfig);
```

Comment:
This may be better as another lifecycle method...not sure what to call it but we need a way for a sources plugins to be able to respond to ***another*** source changing

Reply:
Added `shouldUpdateNamespaceSources` lifecycle method